demos/custom-fields/index.html

```diff
@@ -1,153 +1,54 @@
-
-    <script>
-      function toXml() {
-        var output = document.getElementById('importExport');
-        var xml = Blockly.Xml.workspaceToDom(workspace);
-        output.value = Blockly.Xml.domToPrettyText(xml);
-        output.focus();
-        output.select();
-      }
-
-      function fromXml() {
-        var input = document.getElementById('importExport');
-        var xml = Blockly.Xml.textToDom(input.value);
-        Blockly.Xml.domToWorkspace(xml, workspace);
-      }
-
-      function setRandomStyle() {
-        var blocks = workspace.getAllBlocks();
-        var styles = Object.keys(Blockly.getTheme().getAllBlockStyles());
-        styles.splice(styles.indexOf(blocks[0].getStyleName()), 1);
-        var style = styles[Math.floor(Math.random() * styles.length)];
-        for(var i = 0, block; block = blocks[i]; i++) {
-          block.setStyle(style);
-        }
-      }
-
-      function toggleShadow() {
-        var blocks = workspace.getAllBlocks();
-        for(var i = 0, block; block = blocks[i]; i++) {
-          block.setShadow(!block.isShadow());
-        }
-      }
-
-      function toggleEnabled() {
-        var blocks = workspace.getAllBlocks();
-        for(var i = 0, block; block = blocks[i]; i++) {
-          block.setEnabled(!block.isEnabled());
-        }
-      }
-
-      function toggleEditable() {
-        var blocks = workspace.getAllBlocks();
-        for(var i = 0, block; block = blocks[i]; i++) {
-          block.setEditable(!block.isEditable());
-        }
-      }
-
-      function toggleCollapsed() {
-        var blocks = workspace.getAllBlocks();
-        for(var i = 0, block; block = blocks[i]; i++) {
-          block.setCollapsed(!block.isCollapsed());
-        }
-      }
-
-      function appendDom() {
-        var blocks = document.getElementById('workspace-blocks');
-        if (blocks.firstElementChild) {
-          Blockly.Xml.appendDomToWorkspace(blocks, workspace);
-        }
-      }
-
-      function start() {
-        workspace = Blockly.inject('blocklyDiv', options);
-        appendDom();
-        workspace.scrollCenter();
-      }
-
-      var options = {
-        media: '../../media/',
-        grid: {
-          spacing: 25,
-          length: 3,
-          colour: '#ccc'
-        },
-        move: {
-          scrollbars: true,
-          drag: true,
-          wheel: true,
-        },
-        zoom: {
-          controls: true,
-          startScale: 1.0,
-          maxScale: 4,
-          minScale: 0.25,
-          scaleSpeed: 1.1
-        }
-        /*toolbox: document.getElementById('toolbox')*/
-      }
-
-    </script>
-
-    <xml xmlns="https://developers.google.com/blockly/xml" id="workspace-blocks" style="display: none">
-      <block type="turtle_basic"></block>
-      <block type="turtle_nullifier" y="120"></block>
-      <block type="turtle_changer" y="230"></block>
-    </xml>
-  </body>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Blockly Demos</title>
+  <style>
```

Comment:
This CSS ought to be present in both demos, since it's pretty jarring as the font changes between this page and the two linked pages.